pyproject.toml

```diff
@@ -18,11 +18,18 @@ flask = "^3.0.2"
 pyaml-env = "^1.2.1"
 croniter = "^2.0.3"
 python-simple-logger = "^1.0.19"
-pyhelper-utils = "^0.0.11"
```

Comment:
why this was downgrade?

Reply:
automatically downgraded by poetry, will revert

Reply:
done